scripts/linters/custom_eslint_checks/rules/space-before-opening-brace.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
@Mayank-gaur Eslint already provides a check [space-before-blocks](https://eslint.org/docs/rules/space-before-blocks) but because of the typescript parser we the checks were not working correctly. We had reported this to eslint and had a tracking issue at #10470. It looks like the issue is resolved and once we will update our eslint version this check will start working!

@Hudda, Can you please confirm? Also, do you think we still need the check introduced in this PR?

/cc @sajalasati

Reply:
I think we should make sure if updating the eslint version fixes the issue before closing this PR.

Reply:
I tried testing the existing feature in our existing eslint version (7.1.0) and I can't reproduce any issue:
![image](https://user-images.githubusercontent.com/16653571/113968685-31c99700-9851-11eb-9f4a-58010cf6d3bb.png)


@Hudda Can you please verify? and if that's true then let's close #10470!